src/core/server.h

```diff
@@ -154,6 +158,20 @@ class InferenceServer {
   bool StrictModelConfigEnabled() const { return strict_model_config_; }
   void SetStrictModelConfigEnabled(bool e) { strict_model_config_ = e; }
 
+  // Get / set rate limit mode.
+  RateLimitMode GetRateLimitMode() const { return rate_limit_mode_; }
```

Comment:
We don't typically use "Get". In this case ok since RateLimitMode is an enum... or change to RateLimiterMode()